Gems/GeoJSONSpawner/Code/Include/GeoJSONSpawner/GeoJSONSpawnerBus.h

```diff
@@ -0,0 +1,48 @@
+/**
+ * Copyright (C) Robotec AI - All Rights Reserved
+ *
+ * This source code is protected under international copyright law.  All rights
+ * reserved and protected by the copyright holders.
+ * This file is confidential and only available to authorized individuals with the
+ * permission of the copyright holders.  If you encounter this file and do not have
+ * permission, please contact the copyright holders and delete this file.
+ */
+
+#pragma once
+
+#include <AzCore/EBus/EBus.h>
+
+namespace GeoJSONSpawner
+{
+
+    //! Interface for the GeoJSONSpawner
+    //! Used to spawn, modify and despawn prefabs in runtime
+    class GeoJSONSpawnerRequests : public AZ::ComponentBus
+    {
+    public:
+        using BusIdType = AZ::EntityId;
+
+        //! Spawn prefabs based on the information in provided GeoJSON
+        //! @param rawJsonString - raw string with a GeoJSON to spawn
+        virtual void Spawn(const AZStd::string& rawJsonString) = 0;
```

Comment:
All those calls ins thi API are asynchronous. It is ok, but:
- I cannot see any way to get info on the result.
- What will happen when I call Spawn twice ?